### DISCUSSION TITLE
Problem coming up with SPIEGEL feeds

### DISCUSSION THREAD

**Discussion by bolera** [spacecowboy/Feeder#283] [General]
Hello, I just moved from Flym to Feeder, because I learned that development of Flym was discontinued three years ago. I wanted to report an upcoming problem there and am now trying here because Feeder is affected by the same problem. Also have a few questions about Feeder I'll add below. There is no Issues or did I miss it?

Problem: SPIEGEL has decided to change to some "enhanced" security certificate (as they say) that are not yet supported by some or all RSS readers it seems. When it tries to fetch the article it gets on about 80% the announcement that starting June 6 SPIEGEL will use some kind of "enhanced" security certificate and that I use an obsolete browser that may not support it. Articles with this announcement cannot be fetched already. Looks a bit like they just interpret the UA string and don't really check the capability. don't know. At least I'm unaware of such "enhanced" certificates. There is no mention at their website about this at all. You just get it when you set one of the SPIEGEL feeds in Feeder and want to fetch an article with the Read icon (or whatever that icon is called). Fetching the RSS for the feed is not affected. But may be after June 6. Is there a chance you check what's going on and provide a solution, so that fetching articles within Feeder works again for SPIEGEL (and maybe other sites in the future?)?

That was the problem report. Now to the questions:
1. After fetching an article there is a line starting with a cake symbol and a time like 0:22 oder 0:10 minutes. Not on all feeds it seems. What is that? Is it the download time? It doesn't look to me that it is, but I can't find another explanation. It also cannot be the proposed reading time as I deactivated that.
2. I can't see a difference between the "Article Styles". There are 4 and I checked them out and couldn't really find a difference. e.g. the heading and article information on top of each article takes about half of the screen even in "super compact" mode. I consider this a bit much. Also, it's not clear if this format applies really to the articles or to the article list.
3. There is an option "Nur Namen anzeigen" (translation: show only names) which I cannot make sense of. What names? Is this a wrong translation or what does it actually mean?
4. I installed Feeder because it is described as an Offline Reader. In my understanding an Offline Reader downloads all articles automatically without my intervention. Like lire does (only available for iOS). Is Feeder really doing that or is it fetching each article on a single basis when I press that read icon on top? I'm asking because most of the time it is ok for me that an article is fetched in real-time. However, there are situations where I don't have an internet connection and would like to have the articles offline with me. Typically during a flight. Downloading and saving beforehand is not really an option. (If Feeder is not doing that, is there an rss reader for Android which does? I couldn'd find one yet.)
5. As I said earlier I'm coming from Flym. It also fetches only single articles on command. But it's much easier to command that. You just swish the summary a bit down and that triggers the download and display of the article. That's much more convenient than pointing your finger to the small Read icon on top of the article - where I even have to scroll to sometimes. For instance, if there is a photo and then some longer summary text. It automatically scrolls to the end of that page and thus scrolls the first line off the screen. Would really appreciate if you could provide a similar option as Flym. Btw, recommend a look at Flym. I really like it and would stay with it, if it weren't for the fact that there's no development anymore. I think you should find some other possible enhancements for Feeder in it.

Thanks, and keep up the good work, anyway!
Kai

**Comment by spacecowboy** [comment]
> obsolete browser th

is marketing speak for "you need javascript". Edit the feed in Feeder to open using "custom tab" to get a real browser.


1. it is the reading time. the setting only affects the list.
2. only list. please note the section headers in settings. 
4. what you call the read icon is "fetch full article" (long press icons to get tooltips). and. if you want that automatically, then edit the feed and set that
